src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,71 @@
+package core.basesyntax.service;
+
+import core.basesyntax.exceptions.InvalidUserDataException;
+import core.basesyntax.exceptions.NullUserException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final RegistrationServiceImpl registrationService
+            = new RegistrationServiceImpl();
+
+    @Test
+    void register_validUser_ok() {
+        User validUser = new User("Default", "Default", 18);
+        User outputUser;
+        try {
+            outputUser = registrationService.register(validUser);
+            Assertions.assertEquals(validUser, outputUser, "Register returned wrong User");
+        } catch (InvalidUserDataException e) {
+            Assertions.fail("Wrong data validation");
+        }
```

Comment:
Consider removing the try-catch block in the `register_validUser_ok` test method. Instead, let the test fail naturally if an `InvalidUserDataException` is thrown, as this would indicate that the test case is not passing as expected.